code/tables/0_summary_table.do

```diff
@@ -35,19 +38,52 @@ file open f using "$repository/code/tables/output/summary_table.tex", write repl
 
 // write the beginning of the table
 file write f "\begin{table}[htbp]" _n ///
-"\caption{Title Me Here}" _n ///
+"\caption{Summary Statistics By Host Race: Listing Characteristics}" _n ///
 "\begin{center}%" _n ///
-"\small\begin{tabular}{l c c}" _n
+"\small\begin{tabular}{l c c c c c c}" _n
 
 // write column headers
 file write f " &  & All" _n
 file write f "\\" _n ///
 "\hline\hline\noalign{\smallskip} " _n
 
 // write in stats
-// hi pick up here
-
-
+local ncat price number_of_reviews accommodates bedrooms bathrooms beds amenities cleaning_fee extra_people minimum_nights first_review
```

Comment:
are you sure first_review is an non-cat?i actually dont remember....go back to the code where i create it, it's either a date or just a part of a data.